manage.py

```diff
@@ -9,14 +9,16 @@
 
 import os
 import sys
-
-from django.core.management import execute_from_command_line
+import warnings
 
 from pootle import syspath_override  # noqa
+from pootle.core.log import cmd_log
+
+from django.core.management import execute_from_command_line
```

Comment:
If this import doesn't need to be imported after Pootle ones then it is incorrectly placed. If it needs to be here I will put a comment before stating it.